pandas/core/window.py

```diff
@@ -143,6 +171,21 @@ def __unicode__(self):
         return "{klass} [{attrs}]".format(klass=self._window_type,
                                           attrs=','.join(attrs))
 
+    def _get_index(self, index=None):
+        """
+        Return index & units as ndarrays
+
+        Returns
+        -------
+        tuple of (index, unit)
+        """
+        # return the index
+        if self.is_freq_type:
+            if index is None:
+                index = self._on
+            return index, index.asi8, self.win_unit
+        return index, index, self.win_unit
```

Comment:
This is a little bit opaque. Is the purpose of the 2nd entry in the tuple to ensure you are always passing integers to the Cython function?